timeboost-types/src/bundle.rs

```diff
@@ -377,11 +377,50 @@ impl Transaction {
     }
 }
 
+pub struct SortedTransaction {
```

Comment:
Maybe we can include recovered addresses directly in `Transaction` and drop this `SortedTransaction` type?